data/abilities.ts

```diff
@@ -2357,6 +2357,10 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 					this.add('-end', target, 'Slow Start', '[silent]');
 				}
 			}
+			const strongWeathers = ['desolateland', 'primordialsea', 'deltastream'];
+			if (strongWeathers.includes(this.field.getWeather().id)) {
+				this.field.clearWeather();
```

Comment:
This works; what happens if you send an 'End' event associated with the Pokemon causing the Primal weather event? I would prefer to do it that way rather than just clearing weather directly.

Reply:
Yea that's cleaner. It works too, just changed it